### ISSUE TITLE
Does not work on mac OSX

### ISSUE THREAD

**Issue by X-Ryl669** [carlin-q-scott/browser-media-players#5]
Or I'm missing something ?

**Comment by carlin-q-scott** [comment]
No, you are right. It was reported to Mozilla in this bug comment: https://bugzil.la/1251795#c11

**Comment by porglezomp** [comment]
A workaround for anyone else running into this, I just set the corresponding f-keys as my hotkeys, so I just hold fn when I want to do firefox media stuff.

**Comment by ahsath** [comment]
@porglezomp Hi, I'm sorry where did you change the corresponding f-keys? are we talking about firefox mac?

**Comment by carlin-q-scott** [comment]
@ahsath You can set custom hotkeys in the extension options. It's available through the details page for this extension in the extension manager.

**Comment by psifertex** [comment]
Weird -- https://bugzilla.mozilla.org/show_bug.cgi?id=1591230 (which was created as the child from the link above) reports that the bug is resolved? But I still don't see mediakeys working when I test them on MacOS right now. 

Should that bug be re-opened? Or should it now be possible to support and this add-on just needs updating?

**Comment by carlin-q-scott** [comment]
@psifertex It does look like Global Media Key support for MacOSX was released Monday evening of this week. That is great news!

Do you have the latest Firefox update installed? This extension already supports Global Media Keys through the standardized WebExtension API and they've always worked on Chrome and Opera.

**Comment by psifertex** [comment]
I'm running:
Media Keys Version 2.5.2
Firefox Version 73.0.1 (64-bit)

Do I need to do something else? Still doesn't look like it works.

**Comment by psifertex** [comment]
*ping* Just following up on this again. still does not work on MacOS and I'm on 77.0.1 (64-bit) and 2.5.2 of Media Keys.

**Comment by psifertex** [comment]
For context -- when I press the "play/pause" media key with Firefox focused, iTunes comes up instead. 

Chrome + Streamkeys works as expected. 

I made sure to add both Chrome and Firefox to the Security/Accessibility settings as well, so that's not the issue.
